core/config.py

```diff
@@ -0,0 +1,54 @@
+import os
+from pathlib import Path
+
+import yaml
+
+HOME_PATH = os.path.abspath(os.path.join(os.path.dirname(__file__), os.pardir))
+
+
+class Configuration:
+    ENV = os.environ.get('ENV', default='prod')
+    BASE_URL = 'https://www.cosmosid.com/'
+    LOGIN_PAGE = 'https://app.cosmosid.com/login'
+    BROWSER = os.environ.get('BROWSER', default='firefox')
+
+    config_path = Path.home() / 'PycharmProjects' / 'gl_pro_camp' / 'core'
+    config_path_json = config_path / 'env_configs' / f'{ENV}.yaml'
+    config_path_yaml = config_path / 'config.yaml'
+
+    defaults = {
```

Comment:
I suggest to take defaults from default config file, do not hardcode it here

Reply:
Done.